### PR TITLE
data: bakar-dev

### PR DESCRIPTION
**Pull request by bakar-dev**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6653"></a>

### CODE REVIEW

File: bakar-dev.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+     {
+      "name": "LinkedIn",
+      "url": "https://www.linkedin.com/in/bakardev/",
+      "icon": "FaLinkedin"
+    },
+     {
+      "name": "Github",
+      "url": "https://github.com/bakar-dev",
+      "icon": "FaGithub"
+    },
```
Comment (khushalarora) [inline bakar-dev.json:25]: Also, remove the extra **'Comma'** from this line :+1: 
```Suggestion
    }
```

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hey @bakar-dev, thank you for contributing :tada: 
Please rename your file and Add **`data/`** in the beginning.
Your filename will look like **`data/bakar-dev.json`** :+1:

File: bakar-dev.json (hunk 1 above)
Comment (bakar-dev) [reply bakar-dev.json]: Thanks. I have updated the file.

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks! 🎉 

<p align="center"><a href="https://linkfree.io/docs/how-to-guides/editing"></a></p>

**Pull request merged**